### ISSUE TITLE
Doesn't work with newer version of fabricloader

### ISSUE THREAD

**Issue by sirmac08** [Bstn1802/AutoReconnect#16]
net.fabricmc.loader.impl.FormattedException: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install fabric, version 0.30.0 or later.
Unmet dependency listing:
	 - Mod 'AutoReconnect' (autoreconnect) 1.3.2 requires any version of fabric, which is missing!
Inactive mods:
	 - mod 'Fabric API Base' (fabric-api-base) 0.3.0+a02b446388, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.8+a00e834b88, reason: newer version active
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:188)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:71)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install fabric, version 0.30.0 or later.
Unmet dependency listing:
	 - Mod 'AutoReconnect' (autoreconnect) 1.3.2 requires any version of fabric, which is missing!
Inactive mods:
	 - mod 'Fabric API Base' (fabric-api-base) 0.3.0+a02b446388, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.8+a00e834b88, reason: newer version active
	at net.fabricmc.loader.impl.discovery.ModResolver.findCompatibleSet(ModResolver.java:183)
	at net.fabricmc.loader.impl.discovery.ModResolver.resolve(ModResolver.java:48)
	at net.fabricmc.loader.impl.FabricLoaderImpl.setup(FabricLoaderImpl.java:222)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:186)
	... 3 more

**Comment by Bstn1802** [comment]
Sorry for the late reply
The crash report claims that fabric is missing entirely which doesn't really make sense tbh but maybe you forgot to update the fabric api or that is missing entirely?
What version of fabric are you using and what version of the fabric api? Do you use any other mods?

**Comment by sirmac08** [comment]
I was using the latest version of the API also the version is 0.12.8-1.18 which Idk why it's not working, the other mods that I used were sodium and mod menu that was it and still doesn't work although when I only use the sodium and mod menu it works so that's when I came to a conclusion that it might be the mod itself. Which I still don't know which is the problem.

**Comment by Bstn1802** [comment]
TL;DR Cloth config is required, in case you weren't using it and make sure to use the latest versions of everything and see if that works, it should. I can't reproduce this error :(
I have installed fabric version 0.12.8-1.8 (same as you mentioned). I tested it with only using
- autoreconnect 1.3.2
- cloth-config-5.0.38-fabric
- fabric-api-0.43.1+1.18
- modmenu-3.0.0
Wheras modmenu is optional and cloth config probably is a very old version and any version above that should work too. Not sure about fabric api, but I think its the latest version but again, any possible version above should work as well.

**Comment by sirmac08** [comment]
Turns out I didn't apply the fabric API for some reason nor did I consider it, so it's not the mod's fault it was my own ignorance my bad sorry to bother your time anyways thanks for trying to resolve this problem of mine.
